ancestry_extract.py

```diff
@@ -356,6 +383,8 @@ def get_screenshot(session, target_name):
     """
     Take element screenshot
     """
+    target_name = pathvalidate.sanitize_filepath(target_name)
```

Comment:
Some odd record names where throwing errors. This will make sure that filenames are well formed.